src/styles/main.scss

```diff
@@ -1,3 +1,23 @@
+@import "../utils/variables";
+@import "../blocks/header.scss";
+@import "../blocks/card.scss";
+@import "../blocks/catalog.scss";
+@import "../blocks/stars.scss";
+@import "../blocks/price.scss";
+@import "../blocks/logo.scss";
+@import "../blocks/navigation.scss";
+
+*,
+*::after,
+*::before {
+  box-sizing: border-box;
+  margin: 0;
+}
+
 body {
+  font-family: Roboto, Helvetica, sans-serif;
+}
+
+p {
   margin: 0;
 }
```

Comment:
You are already setting margin 0 in line 10

Reply:
Removed.